packages/web3/src/utils/djb2.ts

```diff
@@ -19,7 +19,7 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 export default function djb2(bytes: Uint8Array): number {
   let hash = 5381
   for (let i = 0; i < bytes.length; i++) {
-    hash = (hash << 5) + hash + (bytes[`${i}`] & 0xff)
+    hash = ((hash << 5) + hash + (bytes[`${i}`] & 0xff)) | 0
```

Comment:
@Lbqds  Why do we need this change? It may break things

Reply:
This is because the result returned by `djb2` is inconsistent with the full node. The full node returns an `int32`, but the previous `djb2` implementation returned an `int64`. So we need to use `| 0` to convert it to `int32`, here is a test that illustrates this issue: https://github.com/alephium/alephium-web3/blob/afe6ed61060828b36194b9418a665765bd0971fa/packages/web3/src/utils/djb2.test.ts#L33